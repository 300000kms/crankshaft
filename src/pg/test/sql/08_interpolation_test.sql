\pset format unaligned
\set ECHO all

WITH a AS (
    SELECT
    ARRAY[800, 700, 600, 500, 400, 300, 200, 100] AS vals,
    ARRAY[ST_GeomFromText('POINT(2.1744 41.403)'),ST_GeomFromText('POINT(2.1228 41.380)'),ST_GeomFromText('POINT(2.1511 41.374)'),ST_GeomFromText('POINT(2.1528 41.413)'),ST_GeomFromText('POINT(2.165 41.391)'),ST_GeomFromText('POINT(2.1498 41.371)'),ST_GeomFromText('POINT(2.1533 41.368)'),ST_GeomFromText('POINT(2.131386 41.41399)')] AS g
)
SELECT (cdb_crankshaft.CDB_SpatialInterpolation(g, vals, ST_GeomFromText('POINT(2.154 41.37)'), 1) - 780.79470198683925288365) / 780.79470198683925288365 < 0.001 FROM a;
