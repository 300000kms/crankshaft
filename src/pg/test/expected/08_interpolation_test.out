SET client_min_messages TO WARNING;
\set ECHO none
<<<<<<< HEAD
 nn  |           nni            |       idw       
-----+--------------------------+-----------------
 200 | 780.79470198683925288365 | 341.46260750526
=======
cdb_spatialinterpolation
t
>>>>>>> ebbf68af5700f33737ac18516e2bdc22ea330b24
(1 row)

